ChangeLog

```diff
@@ -23,6 +23,8 @@ Features
      may be up to two bytes shorter. This allows the library to support all
      hash and signature sizes that comply with FIPS 186-4, including SHA-512
      with a 1024-bit key.
+   * Add key_pwd command line parameter to ssl_server2 and ssl_client2
+     example application to support the use of password-protected key files.
```

Comment:
"application" should probably be "applications".

Reply:
Yes!

Reply:
Fixed.